code/game/gamemodes/miniantags/ftl/enemy_ship/boarding.dm

```diff
@@ -8,12 +8,12 @@
 	var/datum/planet/planet = null
 	var/shield_down
 	var/time_set
-	var/timer = 420 //time before event end (7 minutes)
+	var/timer = 720 //time before event end (7 minutes)
```

Comment:
That's still 7 minutes. Consider that as a fix, since defender got 5 minutes to prepare before shields go down, so technically there is only 7 minutes for event remaining